projects/valtimo/dossier/src/lib/dossier-list/dossier-list.component.html

```diff
@@ -121,9 +121,41 @@ <h3 class="list-header-title">
                 }}</sup>
               </h3>
             </div>
+
+            <div tabs>
```

Comment:
Can you maybe put this in a separate ng-template, to make the template code 'flatter', otherwise it's going to be difficult for future developers to get an overview. So ng-template with ng-container and ngTemplateOutlet